src/components/HeaderComponent/HeaderComponent.jsx

```diff
@@ -47,30 +42,28 @@ const HeaderComponent = () => {
         </picture>
       </NavLink>
       <section className="header__textButtons">
-        <span className="header__homeButton">
-        </span>
         <span className="header__restaurantButton">
           <NavLink to={"/restaurants"}>
-            <b className="header__text">{languageSelector(language, 'headerRestaurant')}</b>
+            <b className="header__text">
+              {languageSelector(language, "headerRestaurant")}
+            </b>
           </NavLink>
         </span>
-        <span className="header__pagesButton">
-        </span>
       </section>
       <section className="header__buttons">
-        <select className="locationList" onChange={handleLocationChange}>
+        <select
+          className="locationList"
+          onChange={handleLocationChange}
+          value={location}
+        >
           <option value={"bogota"}>Bogotá</option>
           <option value={"medellin"}>Medellín</option>
           <option value={"cali"}>Calí</option>
           <option value={"cartagena"}>Cartagena</option>
           <option value={"bucaramanga"}>Bucaramanga</option>
```

Comment:
I consider this options must be generated through looping an array using map, what if you have to add more cities...